packages/components/src/components/domainproperties/dataclasses/DataClassPropertiesPanel.tsx

```diff
@@ -14,12 +14,14 @@ import { getFormNameFromId } from "../entities/actions";
 import { DataClassModel } from "./models";
 import { HelpTopicURL } from "../HelpTopicURL";
 import { DomainPropertiesPanelContext, DomainPropertiesPanelProvider } from "../DomainPropertiesPanelContext";
+import { initQueryGridState } from "../../../global";
 
 const PROPERTIES_HEADER_ID = 'dataclass-properties-hdr';
 const ERROR_MSG = 'Contains errors or is missing required values.';
-
+const DEFAULT_CONTEXT = {collapsed: false};
 const FORM_IDS = {
-    MATERIAL_SOURCE_ID: ENTITY_FORM_ID_PREFIX + 'materialSourceId'
+    CATEGORY: ENTITY_FORM_ID_PREFIX + 'category',
+    SAMPLE_SET_ID: ENTITY_FORM_ID_PREFIX + 'sampleSet'
```

Comment:
SAMPLE_TYPE_ID? in code I've been using the new nomenclature

Reply:
Good catch. Done.